src/components/singlePage/singlePage.css

```diff
@@ -1,3 +1,8 @@
+.mainPage {
+  background-color: lightgray;
+  height: 100vh;
+}
+
```

Comment:
i think u forgot margin-bottom: 35px
also vh didn't do the job as i tried, i used height: 100%

Reply:
that's because there were conflicting css setups in app.css and singlePage.css. That's fixed now. The margin-bottom might be missing, I can't remember what it was for tho

Reply:
im quite sure that app.css was empty before i refilled some stuff, but lets merge it. i will check this finally